src/test/java/com/amazon/ion/conformance/ConformanceTestRunner.kt

```diff
@@ -0,0 +1,77 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+package com.amazon.ion.conformance
+
+import com.amazon.ion.system.*
+import java.io.File
+import org.junit.jupiter.api.DynamicNode
+import org.junit.jupiter.api.TestFactory
+
+object ConformanceTestRunner {
+    val DEFAULT_READER_BUILDER_CONFIGURATIONS = mapOf(
+        "default reader" to IonReaderBuilder.standard()
+            .withCatalog(ION_CONFORMANCE_TEST_CATALOG),
+        "incremental reader" to IonReaderBuilder.standard()
+            .withCatalog(ION_CONFORMANCE_TEST_CATALOG)
+            .withIncrementalReadingEnabled(true),
+        // TODO: Other reader configurations
+    )
+
+    private val DEFAULT_SKIP_FILTER: (File, String) -> Boolean = { file, name ->
+        when {
+            // IonElement can't load $0. TODO: Use IonValue for `produces`, I guess.
+            "$0" in name -> false
+            // For some reason, $ion_symbol_table::null.struct is not handled as expected
+            "IST structs are elided from app view" in name -> false
+            // IonWriter is making it difficult to write invalid data
+            "If no max_id, lack of exact-match must raise an error «then»" in name -> false
+            // IonCatalog's "best choice" logic is not spec compliant
+            // TODO—current test name has a typo. Update to correct spelling once ion-tests is fixed.
+            "When max_id is valid, pad/truncade mismatched or absent SSTs" in name -> false
+            // No support for reading `$ion_encoding` directives yet.
+            "conformance/ion_encoding/" in file.absolutePath -> false
+            // TODO: Seems like the Ion writer is removing "extra" fields from user-supplied symbol tables.
+            file.endsWith("local_symtab_imports.ion") -> when {
+                // If you inspect the debug output, the serialized data does not include the repeated fields.
+                // This implies that the writer is attempting to clean a user-supplied symbol table.
+                "Repeated fields" in name -> false
+                // Unfortunately, the writer also seems to remove "imports" field if and only if
```

Comment:
Oh, right... I started trying to investigate that more, and never finished writing it. Basically, though, the writer will remove `imports: $ion_symbol_table`... presumably because it's not a list of import structs. We should maybe fix this one.

Reply:
+1, that one should be fixed.